Azure VM Linux scenario for demo environments (#150)
microsoft/azure_arc: README.md

* [Connect an existing Windows machine to Azure Arc](azure_arc_servers_jumpstart/docs/onboard_server_win.md)

#### Azure 

> Azure Arc connected machine agent cannot be deployed on Azure VMs as they are already part of Azure Resource Manager. However, for demo purposes, you can use Azure VMs to host an Azure VM and onboard it to Azure Arc. This will allow you to simulate "on-premises" servers on Azure VMs.  

Reviewer: ```suggestion
The following guides in this section will walk you through how to project an Azure VM as an Azure Arc enabled server. 
These guides, using Azure VM as the targeted Azure Arc server are designed **for demo and testing purposes ONLY and are not supported.** 

In each guide, you find a detailed, technical explanation of the mechanism and why it is not expected to project an  Azure VM as an Azure Arc enabled server.
```

> * [Deploy an Ubuntu Azure VM and connect it to Azure Arc using ARM](azure_arc_servers_jumpstart/docs/azure_arm_windows.md)
> 
> * [Deploy a Windows Azure VM and connect it to Azure Arc using ARM](azure_arc_servers_jumpstart/docs/azure_arm_linux.md)

Reviewer: ```suggestion
* [Deploy an Ubuntu Azure VM and connect it to Azure Arc using ARM](azure_arc_servers_jumpstart/docs/azure_arm_template_win.md)

* [Deploy a Windows Azure VM and connect it to Azure Arc using ARM](azure_arc_servers_jumpstart/docs/azure_arm_template_linux.md)
```

> 
>     **Note:** Scenario intended only for demo purposes

Reviewer: ```suggestion
```
